src/views/RecIM/components/List/index.jsx

```diff
@@ -38,9 +42,9 @@ const ParticipantList = ({
 };
 
 const MatchList = ({ matches, activityID }) => {
-  if (!matches?.length) return <Typography>No matches to show.</Typography>;
+  if (!matches?.length || !matches[0]) return <Typography>No matches to show.</Typography>;
```

Comment:
Unfortunately this is not the case. I quite literally added that because it returned me an array with a single null value inside
```
[ null ]
```
hilarious right?